docs/development/agent-prompts/phase-4-worker-preparation.md

```
@@ -0,0 +1,751 @@
+}
+
+export interface DeleteCaseResult {
+  success: boolean;
+}
+
+/**
+ * Financial item payloads
+ */
+export interface AddFinancialItemPayload {
+  caseId: string;
+  itemData: {
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)